css/style.css

```diff
@@ -0,0 +1,155 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Coiny', cursive;
+    font-size: 1rem;
+    line-height: 1.5;
+    color: #AA6373;
+    overflow-x: hidden;
+    width: 100%;
+}
+
+nav {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    width: 100%;
+}
+
+ul {
+    float: right;
+    margin: 25px 20px 0 0;
+
+}
+
+li {
+    list-style: none;
+    display: inline;
+}
+
+a {
+    text-decoration: none;
+    padding: 5px;
+    text-transform: uppercase;
+    color: #AA6373;
+}
+
+li :hover {
+    color: #A9BCD0;
+}
+
+.menu-icon {
+    width:40px;
+    height: 40px;
+    margin: 25px 20px 0 0;
```

Comment:
**PLEASE CHECK THE CONTENT YOURSELF BEFORE DOING MY COMMENT REVIEW BECAUSE I MIGHT BE WRONG**
This line is not affecting the page or any contents.